index.js

```diff
@@ -181,7 +180,11 @@ function expand(str, isTop) {
       N.push(c);
     }
   } else {
-    N = concatMap(n, function(el) { return expand(el, false) });
+    N = [];
+
+    n.map(function(el) {
```

Comment:
```suggestion
    n.forEach(function(el) {
```

since we're not using the return value of `.map`

Reply:
or better (faster) with a regular `for` loop